File: model/src/main/scala/net/server/channel/packet/UseTripleMegaphonePacket.scala
package net.server.channel.packet

class UseTripleMegaphonePacket(private var _position: Short, private var _itemId: Int, private var _lines: Int, private var _message:Array[String], private var _whisper: Boolean) extends AbstractUseCashItemPacket(_position, _itemId) {
  def lines: Int = _lines

  def message: Array[String] = _message

  def whisper: Boolean = _whisper
}

File: engine/engine-model/src/main/java/net/server/channel/packet/cash/use/UseTripleMegaphonePacket.java
package net.server.channel.packet.cash.use;

public class UseTripleMegaphonePacket extends AbstractUseCashItemPacket {
   private final Integer lines;

   private final String[] message;

   private final Boolean whisper;

   public UseTripleMegaphonePacket(Short position, Integer itemId, Integer lines, String[] message, Boolean whisper) {
      super(position, itemId);
      this.lines = lines;
      this.message = message;
      this.whisper = whisper;
   }

   public Integer lines() {
      return lines;
   }

   public String[] message() {
      return message;
   }

   public Boolean whisper() {
      return whisper;
   }
}
